### DISCUSSION TITLE
Attributes for Positional Records' primary constructor

### DISCUSSION THREAD

**Discussion by pinkfloydx33** [dotnet/csharplang#4199] [Q&A] [answered]
Is it possible to add attributes to the constructor of a positional `record`?

The most I've been able to find are ways to add attributes to the type itself, the constructor parameters, the properties and their backing fields:

```csharp
[Baz]
public record Foo(
    [Baz] [field: Baz] [property: Baz] int Bar
);

public class BazAttribute : Attribute {}
```
[SharpLab](https://sharplab.io/#v2:EYLgtghgzgLgpgJwD4AEBMBGAsAKFwbQCEIAvAXVxQGYACBOAYwHsEATGgMSaYApcaBNIqTJCAZgEs4AG1YgaxckIAOCJssQwAnvMWiJAOxgKICXAEoA3Lkq10JkgEEYMBBOABXeDXnPX7rzgaAG8AXxobHAMIMDgoZQgGIJQMADoAJQ8jCVjUgGEmMGUJaUQAZUQANwkkqBD+QWoaewBJKABRAA94BGjpFoMJYzCInFCgA=)

I tried various other targets such as `method` and `constructor`, neither of which are valid.

I'll note that I don't currently have a specific use case for this; however playing around with records and being a reflection-junkie, I found myself curious. Is there some attribute target I'm unaware of?

**Comment by YairHalberstadt** [comment, marked as answer]
discussed at: https://github.com/dotnet/csharplang/discussions/3650

**Reply by pinkfloydx33** [reply]
Bleh, I don't know how I missed that. I guess my search-Foo ain't what it used to be. Thanks!
